views/cypress/tests/properties/simple.spec.js

```diff
@@ -123,20 +124,32 @@
             options.propertyType = propertiesInfo.singleSearchList.type;
             cy.addPropertyToClass(options);
         });
+
+        it('can edit and add new "Calendar" type property to the item class', function () {
+            options.propertyName = propertiesInfo.calendar.name;
+            options.propertyType = propertiesInfo.calendar.type;
+            cy.addPropertyToClass(options);
+        });
+
+        it('can edit and add new "Password" type property to the item class', function () {
+            options.propertyName = propertiesInfo.password.name;
+            options.propertyType = propertiesInfo.password.type;
+            cy.addPropertyToClass(options);
+        });
     });
 
     describe('Child Item Class', () => {
-        it('create a new child item class', function () {
+        before(() => {
+            // Create a new child item class
```

Comment:
Why did we remove this checks?

Reply:
this is setup, not the test

Reply:
Ok, got it, so the scope is only check/assert the properties, but not that the classes and items where created.